src/State/HubInterface.php

```diff
@@ -112,6 +115,18 @@ public function captureEvent(Event $event, ?EventHint $hint = null): ?EventId;
      */
     public function captureLastError(/*?EventHint $hint = null*/): ?EventId;
 
+    /**
+     * Captures a CheckIn to the configured Monitor.
+     *
+     * @param string $slug Identifier of the Monitor
+     * @param MonitorConfig $upsertMonitorConfig Configuration of the Monitor
+     * @param CheckInStatus $status The status of the Monitor
+     * @param CheckIn|null $previos A CheckIn that may have preceded the current CheckIn
+     *
+     * @return CheckIn The created CheckIn
+     */
+    public function captureCheckIn(string $slug, MonitorConfig $upsertMonitorConfig, CheckInStatus $status, ?CheckIn $previous = null): CheckIn;
```

Comment:
This is a breaking change, please remove it.

Reply:
More precisely, we should add the method definition to the PHPDoc of the class using the `@method` tag

Reply:
This was a typo.

Reply:
The typo had nothing to do with the request: a new method cannot be added to an interface without causing a breaking change because that would force all implementors to add it, otherwise their code will not run. That's why I asked to add the method to the PHPDoc: this way we document it and in the next major version of the SDK we will add the method for real.